### ISSUE TITLE
cloudpickle 2.1.0 atribute error

### ISSUE THREAD

**Issue by locotron** [lithops-cloud/lithops#933]
Hello,

I'm running a lithops map_reduce using the last version of cloudpickle, which was installed by lithops. This doesn't happen when using cloudpickle 2.0.0
I got the following error: 

`AttributeError: Can't get attribute '_make_function' on <module 'cloudpickle.cloudpickle' from '/function/cloudpickle/cloudpickle.py'>
`

Code to reproduce the error: 

```
import lithops
def map_fun(obj,my_bucket_name,total_size):
    print(obj.part,my_bucket_name,total_size)
    

def reduce_fun(results):
    print(results)
    
    
if __name__ == "__main__":
    fexec = lithops.FunctionExecutor(
        log_level='DEBUG',runtime='testcontainer',max_workers=1000)
    obj = 's3://some_bucket/data'  # Change-me
    my_bucket_name = 'some_bucket' # Change-me
    total_size = 199999999
    args = {'obj':obj,'my_bucket_name':my_bucket_name,'total_size':total_size}
    
    
    fexec.map_reduce(map_fun, args, reduce_fun,
              obj_chunk_size=1*pow(2,25))
    fexec.wait()
```
If it's useful, i'm using backend aws lambda and storage aws

Thanks.

**Comment by JosepSampe** [comment]
Please provide more details: Lithops version, python version, full debug logs. are you using layers or a container? Do you have the same issue [with this call_async function](https://github.com/lithops-cloud/lithops/blob/master/examples/call_async.py)?

**Comment by locotron** [comment]
Lithops version 2.6
Python version 3.8.10
I'm using a container. A custom runtime image uploaded to aws as a lambda container image.
Doesn't get to the reduce stage, it's within the map, so i will only append the log for the map.
[728009-0-M000.log](https://github.com/lithops-cloud/lithops/files/8793106/728009-0-M000.log)

Yes, it happens with the call_async function example, same error.

**Comment by JosepSampe** [comment]
For me all tests I ran in IBM (not in AWS backends) work ok.

Probably this is an issue of cloudpickle, the only thing we can do in Lithops is to pin the cloudpickle version in the dockerfile template (for aws lambda).

What I experienced some time ago is that newer versions of cloudpickle use to work better with newer versions of python, so probably if you use python3.9 or python 3.10 it will work ok.
